beater/beater.go

```diff
@@ -426,7 +427,8 @@ func (s *serverRunner) run(listener net.Listener) error {
 	cfg := ucfg.Config(*s.rawConfig)
 	parentCfg := cfg.Parent()
 	// Check for an environment variable set when running in a cloud environment
-	if eac := os.Getenv("ELASTIC_AGENT_CLOUD"); eac != "" && s.config.Kibana.Enabled {
+	eac := os.Getenv("ELASTIC_AGENT_CLOUD")
```

Comment:
Is `ELASTIC_AGENT_CLOUD` also set on ECE?

Reply:
yes